x-pack/legacy/plugins/infra/index.ts

```diff
@@ -18,7 +17,7 @@ export function infra(kibana: any) {
     id: APP_ID,
     configPrefix: 'xpack.infra',
     publicDir: resolve(__dirname, 'public'),
-    require: ['kibana', 'elasticsearch', 'metrics'],
+    require: ['kibana', 'elasticsearch', 'metrics', 'observability'],
```

Comment:
With this here, I get:
```
 FATAL  Error: Unmet requirement "observability" for plugin "infra"
```
on start up.

If I remove it, Kibana loads and I see my console logs, but `npSetup.plugins.observability` is undefined.